tests/Adapters/Microsoft.Bot.Builder.Adapters.Twilio.Tests/TwilioAdapterTests.cs

```diff
@@ -22,13 +22,13 @@ public class TwilioAdapterTests
         [Fact]
         public void ConstructorWithTwilioApiWrapperSucceeds()
         {
-            Assert.NotNull(new TwilioAdapter(new Mock<TwilioClientWrapper>(_testOptions).Object));
+            Assert.NotNull(new TwilioAdapter(_testOptions));
         }
 
         [Fact]
-        public void ConstructorShouldFailWithNullTwilioApiWrapper()
+        public void ConstructorShouldFailWithNullOptions()
```

Comment:
The constructor that fails here is the TwilioClientWrapper constructor, not the the Adapter, this test should go in the client and probably not here since the owner of the exception is the other class.